src/pages/About/style.js

```diff
@@ -1,3 +1,69 @@
 import styled from "styled-components";
 
-export const Wrapper = styled.div``;
+export const Wrapper = styled.section`
+    height: auto;
+    width: 90vw;
+    margin: 0 auto;
+    max-height: 863px;
+    background:  linear-gradient(180deg, #4979B7 0%, #3C457B 65.85%, #393A6E 100%), linear-gradient(180deg, #4979B7 0%, #393A6E 100%);
+    border-radius: 25px;
+    color: #FFFFFF;
+    h2 {
+        text-align: center;
+        font-size: 2.5rem;
+        padding: 6rem 0 2.5rem 0;
+    }
+
+    & > p {
+        font-size: 2rem;
+        text-align: center;
+    }
+
+    @media screen and (max-width:768px) {
+        max-height: none;
+    }
+`;
+
+
+export const CardsContainer = styled.div`
+    display: flex;
+    max-width: 100%;
+    padding: 7.5rem 0;
+    justify-content: end;
+
+    @media screen and (max-width:768px) {
+        flex-direction: column;
+        padding: 1rem;
+        margin-top: 1rem;
+        align-items: center;
+    }
+`
+
+export const Card = styled.div`
+    width: 100%;
+    flex-direction: column;
+    padding: 0 3rem;
+
+    p {
+        word-wrap: break-word;
+        text-align: center;
+    }
+
+    & > div {
+        display: flex;
+        justify-content: center;
+        height: 135px;
```

Comment:
É sempre melhor padronizar nossas medidas de altura, margin e padding pra uma medida só, que nesse caso é o rem :)

```suggestion
        height: 6.875rem;
```